index.html

```diff
@@ -335,8 +338,7 @@ <h2><code>PointerEvent</code> Interface</h2>
                     the default value.</p>
                     <p>When the user agent calculates <code>tiltX</code> / <code>tiltY</code> from
                     <code>azimuthAngle</code> / <code>altitudeAngle</code> it SHOULD round the final integer values using
-                    <a href="https://tc39.es/ecma262/#sec-math.round"><code>Math.round</code></a> ([[ECMA-262]]) rules.</p>
-                    <p></p>
+                    <a data-cite="ECMASCRIPT/numbers-and-dates.html#sec-math.round">Math.round</a> ([[ECMASCRIPT]]) rules.</p>
```

Comment:
If it's possible, it might be safer to go with: 
 
```suggestion
                    <a data-cite="ECMASCRIPT#sec-math.round">Math.round</a> ([[ECMASCRIPT]]) rules.</p>
```

In case they move the definition elsewhere in the future.